.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+For urgent operational issues, please contact AWS Support directly.
+https://aws.amazon.com/premiumsupport/
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!-- Not required for feature requests -->
+### Expected Behavior
+
+
+### Observed Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
+for the data we typically require for investigation
+
+GitHub issues are public, please remove sensitive data from logs before posting
```

Comment:
In general agent doesn't log anything truly sensitive (e.g. we'll never log a private crypto key). However, some customers have their own definition of what is sensitive or (in)appropriate for public posting. For example, container instance IDs aren't sensitive, but some people choose to redact them when posting publicly anyway. Same with container image names or task definition names. My recommendation is simply that we remind the customer that the issue is public and that they should take care to post only what they're comfortable posting. They can choose to redact information that they're not comfortable sharing, as long as the message meaning is not lost, or post nothing at all and wait for us to contact them out-of-band.